README.md

```diff
@@ -1,10 +1,14 @@
 # Git Collaboration Practice
 
 ## Overview
+This repo is to help us practice our git workflow.
```

Comment:
Let's add a bit more context to the overview.  Maybe we share the intent behind this project and let other developers know this is just a demo, not a full project.